xml/alerts-2.5.rng

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<grammar xmlns="http://relaxng.org/ns/structure/1.0" 
+         datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
+  <start>
+      <ref name="element-alerts"/>
+  </start>
+
+  <define name="element-alerts">
+    <optional>
+    <element name="alerts">
+      <zeroOrMore>
+        <element name="alert">
+          <attribute name="id"><text/></attribute>
+          <!-- path to the script called for alert -->
+          <attribute name="path"><text/></attribute>
+          <optional>
+            <attribute name="timeout"><text/></attribute>
+          </optional>
+          <optional>
+            <!-- format string fed into strftime defaults to %s -->
+            <!-- use format as with strftime + %N as with date -->
+            <attribute name="tstamp_format"><text/></attribute>
+          </optional>
+          <zeroOrMore>
+            <element name="environment_variables">
```

Comment:
I'm leaning to using the existing "instance_attributes" XML tag instead of adding a new "environment_variables" tag. The main reason is that users are familiar with instance attributes (in the context of resources) as collections of nvpair elements that are not used by pacemaker but passed directly to the (resource agent) script. It's also used for node attributes, so it helps familiarity. Of course they can still be passed as environment variables in practice.

Similarly, I think it would be consistent to use "meta_attributes" for the timeout, timestamp format, etc., again because users are familiar with those being used by pacemaker itself, and pcs already has commands that define instance attributes and meta-attributes, so the new pcs command could follow a similar approach (e.g. pcs alert create <script> [<options>] [meta <options>] recipient <r> ...).